bot/utils/database-migrations.js

```diff
@@ -26,25 +26,21 @@ module.exports.latestMigration = () => {
  * Gets the currently applied migration.
  * @returns {Promise<string>} Promise resolving to the current migration's full name.
  */
-module.exports.current = () => {
-    return db.select("_Migrations", ["value"], {
-        name: "version"
-    }).then(results => {
-        if (results.length === 0) {
+module.exports.current = async () => {
+    const tableCountResult = await db.select("sqlite_master", ["count(*)"]);
```

Comment:
Doesn't this need some form of error handling? What if the database couldn't connect or something like that?

(As a caller, I wouldn't expect a "get current migration" method to raise errors)

Reply:
:thinking: SQL is scary... What if the `_Migrations` table got swallowed by a bit flip?? *Think of the possibilities!*